test/test-helper.ts

```diff
@@ -211,3 +211,32 @@ export class TestDriver {
 
 export const onWindows = process.platform === "win32" ? it : it.skip;
 export const onPosix = process.platform !== "win32" ? it : it.skip;
+
+class Query {
```

Comment:
Just an idea: You could also add a `toString` method (which returns the stack string value) and always return a Query from `driver.synthesizedStack()` calls – that could remove the need for `$`. Would be a breaking change however (type-wise at least)

Reply:
That's a great idea!